Description: Separate check from install so checks can be made optional
 In order to support DEB_BUILD_OPTIONS=nocheck cleanly the install and
 check targets are best separated.
 Also allow COPTS, LDFLAGS and CC to be overridden by environment
 (Kel Modderman <kel@otaku42.de>).
. 
Author: Wookey <wookey@linaro.org>
Last-Update: 2012-04-09

--- a/Makefile
+++ b/Makefile
@@ -25,20 +25,20 @@ else
 ifeq ($(ARCH),i386)
 	  COPTS = -g -O3 -mcpu=i586 -mtune=i686
 else
-	  COPTS = -g -O2
+	  COPTS ?= -g -O2
 endif
 endif
 	 CFLAGS = -W -Wall $(COPTS) $(DEBUG) $(LOOPS) -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
 		  $(ISSUSE) -DINITDIR=\"$(INITDIR)\" -DINSCONF=\"$(INSCONF)\" -pipe
 	  CLOOP = -falign-loops=0
-	LDFLAGS = -Wl,-O,3,--relax
+	LDFLAGS ?= -Wl,-O,3,--relax
 	   LIBS =
 ifdef USE_RPMLIB
 	 CFLAGS += -DUSE_RPMLIB=1
 	LDFLAGS += -Wl,--as-needed
 	   LIBS += -lrpm
 endif
-	     CC = gcc
+	     CC ?= gcc
 	     RM = rm -f
 	  MKDIR = mkdir -p
 	  RMDIR = rm -rf
@@ -134,7 +134,7 @@ else
 	tests/common
 endif
 
-install:	$(TODO) check
+install:	$(TODO)
 	$(MKDIR)   $(SBINDIR)
 	$(MKDIR)   $(SDOCDIR)
 	$(MKDIR)   $(CONFDIR)
